vmsdk/python/cctrusted_vm/sdk.py

```diff
@@ -86,9 +86,14 @@ def get_measurement(self, imr_select:[int, int]) -> TcgIMR:
         if algo_id is None or algo_id is TcgAlgorithmRegistry.TPM_ALG_ERROR:
             algo_id = self._cvm.default_algo_id
 
-        return self._cvm.imrs[imr_index].digest(algo_id)
-
-    def get_quote(self, nonce: bytearray, data: bytearray, extraArgs=None) -> Quote:
+        return self._cvm.imrs[imr_index]
+
+    def get_quote(
```

Comment:
why change to this style, looks wierd.

Reply:
After adding default values for parameters "nonce" and "bytearray", the line became too long. So I followed the https://google.github.io/styleguide/pyguide.html#3192-line-breaking 3.19.4 Default Values for that style.